qcfractal/queue/parsl_adapter.py

```diff
@@ -90,14 +90,17 @@ def count_running_workers(self) -> int:
                 # Efficiency loop break
                 break
 
+        found_readable = False
         for executor_key, executor in self.client.executors.items():
             if hasattr(executor, 'connected_workers'):
                 # Should return an int
                 running += executor.connected_workers
+                found_readable = True
             elif hasattr(executor, 'max_threads') and executor_running_task_map[executor_key]:
                 running += 1
-            else:
-                raise NotImplementedError("Cannot accurately estimate consumption from executors")
+                found_readable = True
+        if not found_readable:
```

Comment:
Just to double check, this will not actually throw an error correct and is caught in the manager?

Reply:
Yes, the only call to that is in the `QueueManager.update` function and it does this:

```python
        try:
            running_workers = self.queue_adapter.count_running_workers()
            log_efficiency = True
        except NotImplementedError:
            running_workers = 0
            log_efficiency = False
```